fixtures/features/swagger/body-schema-example.yaml

```diff
@@ -19,6 +19,7 @@ paths:
         200:
           description: response description
           schema:
+            type: object
```

Comment:
Type was missing so this example fixture doesnt' generate data structure before.